source/production/F0.Minesweeper.Components/Pages/Game/Modules/Cell.razor.cs

```diff
@@ -125,22 +111,13 @@ private bool TryUpdateStatus(CellInteractionType inputCommand, bool? isMine = nu
 
 			CellStatusType newStatus = statusManager.MoveNext(inputCommand, isMine);
 
-			CellStatusTranslation translation = GetTranslationOrDefault(newStatus);
-			StatusText = translation.GetDisplayValue(adjacentMineCount);
-			CssClass = translation.CssClass.ToString();
+			Debug.Assert(visualisationManager is not null, $"The '{nameof(visualisationManager)}' is injected on Cell generation.");
```

Comment:
The ultimate nit-picking: we could "nameof" the `Cell` in the interpolated string.

Reply:
And you know that it's because of this reason that I changed it to your suggestion 😁

Reply:
🤣